quix-frontend/client/src/lib/sql-autocomplete/adapter/sql-autocomplete-adapter.ts

```diff
@@ -62,35 +82,70 @@ export class SqlAutocompleter implements IAutocompleter {
    * @param {TableInfo[]} tables
    * @return {Promise<ICompleterItem[]>}
    */
+
   private async getQueryContextColumns(tables: TableInfo[]) {
     const columnsNamesMemory: Set<string> = new Set();
     const columnsWithPrefixMemory: Set<string> = new Set();
     const tablesPromises: Promise<TableInfo>[] = [];
-
```

Comment:
We like new lines that orginize code structure and seperate between parts

Reply:
Usually you should have an enter before+after a condition/loop (if/for/while etc) and before a return line.